.github/workflows/gradle.yml

```diff
@@ -51,20 +47,49 @@ jobs:
         rm -rf build/libs/*.jar
         npm ci
         npx semantic-release
-      if: matrix.gradle == '7.0'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Run SonarQube Scanner
       run: |
         if [ "$SONAR_LOGIN" != "" ]; then
           ./gradlew sonarqube -Dsonar.login=$SONAR_LOGIN --no-daemon
         fi
-      if: matrix.gradle == '7.0'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         SONAR_LOGIN: ${{ secrets.SONAR_LOGIN }}
     - uses: actions/upload-artifact@v2
       if: always()
       with:
-        name: reports (Gradle ${{ matrix.gradle }})
+        name: reports
+        path: build/reports
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        gradle: ['7.0', '7.3.1']
+        os: ['ubuntu-latest', 'windows-latest']
+        exclude:
+          # used in the sonar-and-semantic-release job
+          - gradle: '7.3.1'
+            os: 'ubuntu-latest'
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'temurin'
+        java-version: 11
+        cache: gradle
+    - name: Build with Gradle
+      env:
+        SIGNING_KEY: ${{ secrets.SIGNING_KEY }}
+        SIGNING_PASSWORD: ${{ secrets.SIGNING_PASSWORD }}
+      run: |
+        ./gradlew build --no-daemon "-Dsnom.test.functional.gradle=${{ matrix.gradle }}"
+    - uses: actions/upload-artifact@v2.2.4
```

Comment:
This version is necessary to avoid a known problem in the latest release.
Refs https://github.com/actions/upload-artifact/issues/281